src/actuator/UtilitiesActuator.C

```diff
@@ -71,6 +72,79 @@ double Gaussian_projection(
   return g;
 }
 
+void resize_std_vector(
```

Comment:
Some of this functionality has been implemented in `ElemDataRequests` and `ScratchViews`. It would be good to revisit this and see if we can minimize the overlap in functionality.

Reply:
Definitely. Will do.